conf/drupal/config/core.entity_form_display.taxonomy_term.tags.default.yml

```diff
@@ -0,0 +1,34 @@
+uuid: 75e84974-e47d-41fc-9577-089f8e6f4353
+langcode: en
+status: true
+dependencies:
+  config:
+    - taxonomy.vocabulary.tags
```

Comment:
I'll remove this. Oddly enough, it was not a changed file, but a completely newly added one.